data/logs_model/__init__.py

```diff
@@ -26,9 +26,7 @@ def __init__(self):
 
     def initialize(self, model):
         self._model = model
-        logger.info("===============================")
-        logger.info("Using logs model `%s`", self._model)
-        logger.info("===============================")
+        logger.debug("Using logs model `%s`", self._model)
```

Comment:
Leave this and the next one as info

Reply:
@josephschorr Do you want to include those log lines which only include `=` characters? If so, want me to just make one line string and include line breaks?